roles/cinder-common/tasks/ceph_integration.yml

```diff
@@ -1,4 +1,12 @@
 ---
+- name: Install ceph-common
+  apt: pkg={{ item }} state=latest
```

Comment:
I would suggest to specify ceph-common version instead of installing latest version.
So that we can keep ceph server and ceph client at the same version number.
If ceph client have higher version than ceph server, new features of ceph client may be not supported by ceph server.